frontend/server/libs/dao/Users_Badges.dao.php

```diff
@@ -25,4 +25,17 @@ public static function getUserOwnedBadges(Users $user) {
         $args = [$user->user_id];
         return $conn->GetAll($sql, $args);
     }
+
+    public static function getUserBadgeAssignationTime(Users $user, string $badge): ?int {
+        global $conn;
+        $sql = 'SELECT
+                    UNIX_TIMESTAMP(ub.assignation_time) AS assignation_time
+                FROM
+                    Users_Badges ub
+                WHERE
+                    ub.user_id = ? AND ub.badge_alias = ?;';
+        $args = [$user->user_id, $badge];
+        $rs = $conn->getRow($sql, $args);
```

Comment:
argh, se me olvidó esto ><

```suggestion
        return $conn->getOne($sql, $args);
```

Reply:
Done